frontend/src/metabase/parameters/components/ParameterLinkedFilters/ParameterLinkedFilters.tsx

```diff
@@ -38,27 +38,107 @@ const ParameterLinkedFilters = ({
   onChangeFilteringParameters,
   onShowAddParameterPopover,
 }: ParameterLinkedFiltersProps): JSX.Element => {
-  const [expandedParameterId, setExpandedParameterId] = useState<ParameterId>();
-
-  const filteringParameters = useMemo(
-    () => parameter.filteringParameters ?? [],
-    [parameter],
-  );
-
   const usableParameters = useMemo(
     () => otherParameters.filter(usableAsLinkedFilter),
     [otherParameters],
   );
 
+  return (
+    <SectionRoot>
+      <SectionHeader>{t`Limit this filter's choices`}</SectionHeader>
+      <Content
+        usableParameters={usableParameters}
+        parameter={parameter}
+        onChangeFilteringParameters={onChangeFilteringParameters}
+        onShowAddParameterPopover={onShowAddParameterPopover}
+      />
+    </SectionRoot>
+  );
+};
+
+function Content({
```

Comment:
BTW, we also agreed that we want to [break large files down](https://www.notion.so/56f6f59da3b647de89acb28ea9ebc424?v=cca33933b6244a7a94f256d1c110ce09&p=536776c6ff98451aa1cd3c2b130b08d7&pm=s), i.e. have component definitions in separate files.

Reply:
> have component definitions in separate files

That seems like a pretty extreme reaction to the problem of "we have some large files that are hard to understand". I've never seen anyone advocate for one-function-per-file, and this seems like very similar guidance. I'll try it though, but in a separate PR.